mp/game/momentum/base.fgd

```diff
@@ -7524,20 +7524,29 @@
 ]
 
 // lightprop("models/editor/spot.mdl") <---- use this once the orientation is unfucked
-@PointClass base(Targetname, Parentname, Angles) size(-2 -2 -2, 2 2 2) frustum(lightfov,nearz,farz,lightcolor,-1) = env_projectedtexture : 
```

Comment:
Should this size have been removed?

Reply:
The size here made it appear as a 4x4x4 box in hammer and now it's a model.